Mobile Buy SDK/Mobile Buy SDK/Models/BUYManagedObject.h

```diff
@@ -25,8 +25,8 @@
 //
 
 #import <CoreData/CoreData.h>
-
-#import <Buy/BUYObject.h>
+#import "BUYObject.h"
```

Comment:
For consistency. We always use local-file `#import` style aside from several cases (such as this one). Using the framework-style `#import` also requires that the header is `public` and exposed in the umbrella header, which we may or may not want to do for some files. Using the quotation mark import requires no changes if we decided to go either way.